src/components/grape-browser/GrapeBrowser.js

```diff
@@ -266,8 +273,6 @@ export default class GrapeBrowser extends Component {
     const isBrowserOpened = Boolean(this.state.browser)
     const hasTrigger = Boolean(query && query.trigger)
 
-    clearTimeout(this.searchBrowserInputTimeoutId)
```

Comment:
Why is this not needed anymore?

Reply:
sorry, deleted this by accident. thanks!